### ISSUE TITLE
Can't open dialogs that read a configobj config.ini

### ISSUE THREAD

**Issue by Komodovaran** [mherrmann/fbs#108]
I store a config.ini in the resources part of the app, which I read and write to using the package `configobj` with absolute path hacks to make the .app GUI run. However, it silently fails (keyboard shortcut registered in menubar, but window doesn't open - no crashing though), and all file dialogs silently fail as they presumably fail to read the config.ini.

**Comment by mherrmann** [comment]
Why absolute paths and not [`get_resource(...)`](https://build-system.fman.io/manual/#resource-files)? Have you tried `freeze --debug`, described in https://build-system.fman.io/troubleshooting?

**Comment by Komodovaran** [comment]
Whoops, I hadn't realized they did the exact same thing. I followed the method in https://stackoverflow.com/questions/404744/determining-application-path-in-a-python-exe-generated-by-pyinstaller.

Anyway, I've played around a bit, and I think the error is actually the .app being unable to write to the config.ini, as the contents are being read just fine if I display them directly in the GUI. It doesn't matter if the config.ini is located inside the .app or outside.
